src/pybind/mgr/dashboard/frontend/src/app/shared/pipes/list-to-html.pipe.spec.ts

```diff
@@ -0,0 +1,31 @@
+import moment from 'moment';
+
+import { ListToHtmlPipe } from './list-to-html.pipe';
+
+describe('ListToHtmlPipe', () => {
+  const pipe = new ListToHtmlPipe();
+
+  it('create an instance', () => {
+    expect(pipe).toBeTruthy();
+  });
+
+  it('transforms "list to html"', () => {
+    const date: Date = moment().subtract(65, 'minutes').toDate();
+    expect(
+      pipe.transform([
+        {
+          created: date,
+          subject: 'service:prometheus',
+          level: 'INFO',
+          message: 'service was created'
+        }
+      ])
+    ).toBe(
+      '<b>An hour ago</b> - <span class="badge badge-info" ng-reflect-ng-class="badge-info">service:prometheus</span><br><i class="fa fa-info-circle indent" aria-hidden="true"></i><p class="indent">service was created</p><br>'
```

Comment:
This is an unusally long line. Can this be split into small chunks?

Reply:
I tried this as well. The problem is if we move the content to next line..it creates an extra space in the expected value which results in the failing of test.

Reply:
Makes sense. Thanks

Reply:
@aaSharma14 you sure you cannot do this?
```js
'<b>An hour ago</b> - <span class="badge badge-info" ng-reflect-ng-class="badge-'info">'
+ 'service:prometheus</span><br><i class="fa fa-info-circle indent" aria-hidden="true"></i>'
+ '<p class="indent">service was created</p><br>'
```

In any case, I think this is the proof that we shouldn't mix program flow and data processing with presentation. There's no much value on validating that the HTML chunk is exactly like this.